azure_pipeline.yaml

```diff
@@ -6,18 +6,164 @@ trigger:
+      storage_account_rg: 'core-infra-ldata-rg'
+      storage_account_name: 'ethosldata'
+      dependsOn: 'sbox_cftapps_cluster_lb'
+
+    - deployment: 'ldata_cftapps_cluster_lb_backend'
+      environment: 'ldata'
+      component: 'cftapps_cluster_lb_backend'
+      service_connection: 'azurerm-ethosldata'
+      storage_account_rg: 'core-infra-ldata-rg'
+      storage_account_name: 'ethosldata'
+      dependsOn: 'sbox_cftapps_cluster_lb'
+
+    - deployment: 'demo_cftapps_cluster_lb_backend'
+      environment: 'demo'
+      component: 'cftapps_cluster_lb_backend'
+      service_connection: 'dcd-cftapps-demo'
+      storage_account_rg: 'core-infra-demo-rg'
+      storage_account_name: 'cftappsdemo'
+      dependsOn: 'sbox_cftapps_cluster_lb'
+
+    - deployment: 'test_cftapps_cluster_lb'
+      environment: 'test'
+      component: 'cftapps_cluster_lb'
+      service_connection: 'dcd-cftapps-test'
+      storage_account_rg: 'core-infra-test-rg'
+      storage_account_name: 'cftappstest'
+      dependsOn: 'sbox_cftapps_cluster_lb'
+
+    - deployment: 'test_cftapps_cluster_lb_backend'
+      environment: 'test'
+      component: 'cftapps_cluster_lb_backend'
+      service_connection: 'dcd-cftapps-test'
+      storage_account_rg: 'core-infra-test-rg'
+      storage_account_name: 'cftappstest'
+      dependsOn: 'sbox_cftapps_cluster_lb'
+
+    - deployment: 'stg_cftapps_cluster_lb'
+      environment: 'stg'
+      component: 'cftapps_cluster_lb'
+      service_connection: 'dcd-cftapps-stg'
+      storage_account_rg: 'core-infra-stg-rg'
+      storage_account_name: 'cftappsstg'
+      dependsOn: 'sbox_cftapps_cluster_lb'
+
+    - deployment: 'stg_cftapps_cluster_lb_backend'
+      environment: 'stg'
+      component: 'cftapps_cluster_lb_backend'
+      service_connection: 'dcd-cftapps-stg'
+      storage_account_rg: 'core-infra-stg-rg'
+      storage_account_name: 'cftappsstg'
+      dependsOn: 'sbox_cftapps_cluster_lb'
+
+    - deployment: 'prod_cftapps_cluster_lb'
+      environment: 'prod'
+      component: 'cftapps_cluster_lb'
+      service_connection: 'dcd-cftapps-prod'
+      storage_account_rg: 'core-infra-prod-rg'
+      storage_account_name: 'cftappsprod'
+      dependsOn: 'sbox_cftapps_cluster_lb'
+
+    - deployment: 'prod_cftapps_cluster_lb_backend'
+      environment: 'prod'
+      component: 'cftapps_cluster_lb_backend'
+      service_connection: 'dcd-cftapps-prod'
+      storage_account_rg: 'core-infra-prod-rg'
+      storage_account_name: 'cftappsprod'
+      dependsOn: 'sbox_cftapps_cluster_lb'
+
+    - deployment: 'prod_shutter' 
+      environment: 'prod'
+      component: 'shutter'
+      service_connection: 'dcd-cftapps-prod'
+      storage_account_rg: 'core-infra-prod-rg'
+      storage_account_name: 'cftappsprod'
+      dependsOn: 'sbox_cftapps_cluster_lb'
+
 stages:
   - stage: Precheck
     jobs:
       - job: Validate
-        pool:
-          vmImage: $(agentPool)
+        pool:   
```

Comment:
```suggestion
        pool:
```

Reply:
vmImage is required if we want to keep as it is with the reference to Microsoft hosted image:- ubuntu-18.04

If just using pool, it fails, 
"The pipeline is not valid. Could not find a pool with name ubuntu-18.04"